src/servicetask/servicetaskenvironment.ts

```diff
@@ -52,7 +52,7 @@ export interface IServiceTaskRoxApi {
   getEfApiToken(): Promise<string>;
   getApiToken(): string;
   getRoxtraTokenByUserId(userId: string): Promise<string>;
-  getSupervisor(userId: string): Promise<string>;
+  getSupervisor(userId: string): Promise<{ type: string; value: string | number }>;
```

Comment:
Declare the type as a literal type. Like { type: "group" | "user" | "error", ... }
Then the comiler knows that this property can only have these values.